README.md

```diff
@@ -1117,9 +1117,9 @@ If `url` is:
 
 ### What is the `ixlib` Param on Every Request?
 
-For security and diagnostic purposes, we tag all requests with the language and version of library used to generate the URL.
+For security and diagnostic purposes, we tag all requests with the language and version of library used to generate the URL. To disable this, we provide two options. For disabling the `ixlib` parameter across all requests, we provide `disableIxlibParam` as a plugin option for use in `gatsby-config.js`.
```

Comment:
Maybe it's worth adding a little example code here as we do below for `indludeLibraryParam`?

Reply:
Agreed, added to the PR.